README.md

```diff
@@ -1 +1,79 @@
-# configuration-caas
+# Cluster-as-a-Service base configuration
+
+This repo provides a configuration for a Cluster-as-a-Service, built on top of
+Crossplane. This configuration exposes a simple API to your internal developers
+for creating Kubernetes clusters. In the background, this configuration can create clusters in AWS, Azure, and GCP. It automatically applies common recommended practices, such as connecting the cluster to existing Flux deployments, automating VPC setup, and creating a Flux operator in a cluster.
+
+This repo is a starting point for you to deliver your own
+Cluster-as-a-Service. Fork this repository and customize the configuration to
+meet your teams' needs.
+
+To deploy a multicloud CaaS control plane in your organization, check out the
+[Multicloud Quickstart](https://docs.upbound.io/quickstart/multicloud-deploy/).
+
+Advantages of CaaS:
+
+- GitOps workflow
+- Production-ready template
+- Scalable architecture
+- Product agnostic approach
+
+## Available APIs
+
+This repository uses Compositions for AWS, Azure, and GCP provider APIs, as well as the Upbound Control Plane provider. For more information, review the provider Configurations below:
```

Comment:
>This repository uses

Instead of uses, what do you think about "defines" or "implements"? That feels more accurate to me.

>review the provider Configurations below

The links below aredocumentation for each of the APIs. What do you think about instead "review the API documentation below"?